docs/admin/predeployment_check.md

```diff
@@ -108,14 +120,12 @@ with all the relevant settings.
 
 For more information, please refer to the [librdmacm documentation](https://github.com/linux-rdma/rdma-core/blob/master/Documentation/librdmacm.md)
 
-### Subnet
-
-Since all engines need to be able to communicate, the different network
```

Comment:
This does not make sense to me. Maybe:
Since all engines need to communicate, the different network interfaces must be on the same subnet or you must configure the routing capabilities across the different subnets